### ISSUE TITLE
There is no language input field in the Multi-File window, is it right?

### ISSUE THREAD

**Issue by ash0080** [ZeroX-DG/SnippetStore#66]
![2020-10-22_11-20-33 (1)](https://user-images.githubusercontent.com/2803210/96820467-a2722800-1458-11eb-805c-d3ee8b636478.png)

**Comment by ZeroX-DG** [comment]
Yes, this is a design mistake. For now you can only specify the language using the file extension. For example, if the file name end with `.js` you will get `JavaScript` highlighting.
